ktor-client/ktor-client-cio/jvm/test/io/ktor/client/engine/cio/CIORequestTest.kt

```diff
@@ -55,6 +51,7 @@ class CIORequestTest : TestWithKtor() {
         }
     }
 
+    @CoroutinesTimeout(10_000)
```

Comment:
Could we use class-level annotation instead?
https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-debug/kotlinx.coroutines.debug.junit5/-coroutines-timeout/

Reply:
Yeah I originally had moved it to the class annotation but it was timing out because it applies for the execution of the whole collection of tests.  I can just sum them up and use the one annotation.